src/condor_credd/condor_credmon_oauth/credmon/CredentialMonitors/LocalCredmon.py

```diff
@@ -58,7 +62,16 @@ def refresh_access_token(self, username, token_name):
         token.update_claims({'sub': username})
         user_authz = self.authz_template.format(username=username)
         token.update_claims({'scope': user_authz})
-        token.update_claims({'ver': 'scitokens:2.0'})
+
+        # Only set the version if we have one.  No version is valid, and implies scitokens:1.0
+        if self.token_ver:
+            token.update_claims({'ver': self.token_ver})
+
+        # Convert the space separated list of audiences to a proper list
+        # No aud is valid for scitokens:1.0 tokens.  Also, no resonable default.
+        aud_list = self.token_aud.strip().split(" ")
```

Comment:
I bet we would be fine to use `.split()` here... do we care if they have multiple whitespace delimiters?
```suggestion
        aud_list = self.token_aud.strip().split()
```